keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -312,30 +302,43 @@ proposed resize operation for a given resource.  Any time the
 `Pod.Status.ContainerStatuses[i].Resources` field, this new field explains why.
 
 This field can be set to one of the following values:
-* `Proposed` - the proposed resize (in Spec...Resources) has not been accepted or
-  rejected yet. Desired resources != Allocated resources.
-* `InProgress` - the proposed resize has been accepted and is being actuated. A new resize request
-  will reset the status to `Proposed`. Desired resources == Allocated resources != Actual resources.
+* `InProgress` - the proposed resize has been accepted and is being actuated. A `Deferred` or
+  `Infeasible` resize will take precedence over `InProgress`.
```

Comment:
Please consider dropping this if we can respond infeasibility in the context of /resize. Unless I missed something, we can determine infeasibility for bound pods inline during resize request. For unscheduled pods, the scheduler will  issue a "not enough..."

Reply:
We should move some of these checks to admission-time for the sake of failing fast, but I think we still need the infeasible state. At least for now, we need it to reject pods with a static CPU policy, but I could also imagine needing it in the future when/if we expand the resources that can be resized.